src/index.js

```diff
@@ -0,0 +1,141 @@
+"use strict";
+
+const state = {
+  city: 'Portland, OR',
+  temperature: 72,
+  latitude: 45.5152,
+  longitude: 122.6784,
+};
+
+let getCityName;
+let curWeatherEmojis = document.getElementById("curWeatherEmojis");
+let cityDisplayOnLoad = document.getElementById("now-showing").textContent = 
+  `Now showing the temperature for ${state.city}`;
+
+const updateCityWhileTyping = () => {
+  const input = document.getElementById("defaultCity").value;
+  document.getElementById("now-showing").textContent = 
+  `Now showing the temperature for ${input}`
+  state.city = input;
+};
+
+document.getElementById("sky").addEventListener('change', (event) => {
+  const bgValue = event.target.options[event.target.selectedIndex].value;
+  if( bgValue == 'sunny' ) {
+    document.body.style.backgroundImage = "url(/ada-project-docs/assets/sunshine.png)";      
+    } else if( bgValue == "cloudy" ){
+      document.body.style.backgroundImage = "url(/ada-project-docs/assets/cloudy.png)";      
+    } else if( bgValue == "rainy" ){
+      document.body.style.backgroundImage = "url(/ada-project-docs/assets/rain.png)";      
+    } else if( bgValue == "snowy" ){
+      document.body.style.backgroundImage = "url(/ada-project-docs/assets/snow.png)";      
+    }
+});
+
+const weatherEmojisandColor = () => {
+  let temp = state.temperature;
+  let color;
+  if (temp <= 49) {
+    curWeatherEmojis.textContent="❄️🥶❄️🥶";
+    document.body.style.backgroundImage = "url(/ada-project-docs/assets/snow.png)";
+    color="cornflowerblue";
+  } else if (temp <= 59) {
+    curWeatherEmojis.textContent="🧤🧣🧤🧣";
+    document.body.style.backgroundImage = "url(/ada-project-docs/assets/rain.png)";
+    color = "green";
+  } else if (temp <= 69) {
+    curWeatherEmojis.textContent="✅✅✅✅";
+    document.body.style.backgroundImage = "url(/ada-project-docs/assets/cloudy.png)";
+    color ="yellow";
+  } else if (temp <= 79) {
+    curWeatherEmojis.textContent="😅😎😅😎";
+    document.body.style.backgroundImage = "url(/ada-project-docs/assets/sunshine.png)";
+    color="orange";
+  } else {
+    curWeatherEmojis.textContent="🔥🔥🔥🔥";
+    document.body.style.backgroundImage = "url(/ada-project-docs/assets/dry.png)";
+    color="red";
+  }
+    let tempTag = document.getElementById("tempTag");
+    
+    tempTag.textContent = String(temp);
```

Comment:
another way to create a string, without using the String constructor is to just use a template string
```
`${state.temp}`